main.go

```diff
@@ -0,0 +1,38 @@
+/*
+ *Author: Stefan
+ *Date: 11/20/2019
+ *Last changes: 11/20/2019 15.33
+ *Task: Write reverse([]int64) []int64 function that returns the copy of the original slice in reverse order. The type of elements is int64.
+ *						Input -> (1, 2, 5, 15)
+ *						Output -> (15, 5, 2, 1)
+**/
+
+package main
+
+import "fmt"
+
+//reverse function
+func reverse(sl []int64) []int64 {
+	var n = len(sl)
+	var c = cap(sl)
+	//Just make copy with original len and cap
+	var cpSl = make([]int64, n, c)
```

Comment:
Do we really need separate variables `n` and `c` ?
Will be better to write smth like: `make([]int64, len(sl), cap(sl))`

Reply:
Yes, correct.:) Done